src/lib/assets/holidayLists.ts

```diff
@@ -0,0 +1,21 @@
+export type HolidayData = {
+	name: string;
+	date: Date;
+	words: string[];
+	colors: [string, string];
+};
+
+export const holidayLists: HolidayData[] = [
+	{
+		name: "New Year's Day",
+		date: new Date(2022, 0, 1),
```

Comment:
this should be 2023, right?

Reply:
We don't use the year since these will loop every year. Only reason we put it here is since date requires a year.

Reply:
I will make the change anyway.

Reply:
ooh, you right